src/Components/QuickTimesDropdown/QuickTimesDropdown.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+    IQuickTimesDropdownProps,
+    IQuickTimesDropdownStyleProps,
+    IQuickTimesDropdownStyles
+} from './QuickTimesDropdown.types';
+import { getStyles } from './QuickTimesDropdown.styles';
+import {
+    classNamesFunction,
+    useTheme,
+    styled,
+    IDropdownOption,
+    Dropdown
+} from '@fluentui/react';
+import { TFunction } from 'i18next';
+import { QuickTimeSpans } from '../../Models/Constants/Constants';
+import { useTranslation } from 'react-i18next';
+
+const getClassNames = classNamesFunction<
+    IQuickTimesDropdownStyleProps,
+    IQuickTimesDropdownStyles
+>();
+
+const QuickTimesDropdown: React.FC<IQuickTimesDropdownProps> = (props) => {
```

Comment:
It's not very standard to have a dropdown inside a context menu. The recommended/standard pattern for context menus is to have a nested menu like this. That should come natively in context menu if simply pass in the `subMenuProps` item to the context menu item.

![image](https://user-images.githubusercontent.com/57726991/197848731-dcd9a64a-fca5-49df-95a7-eed204bff4df.png)


![image](https://user-images.githubusercontent.com/57726991/197848537-a8ebe3c4-53b6-4541-a12c-b4dcdbc66715.png)

Reply:
yep I am aware of that prop and my initial commit was with using it, but since Matt talked about it to be a dropdown that is why I made a standalone component that we can consume both in the builder and viewer. @darsney do you have a preference here?

Reply:
since we opted to have it not in the menu we can ignore this now.